lib/discovery_dispatcher/indexing_job.rb

```diff
@@ -20,13 +21,15 @@ def perform
       Rails.logger.info "Completed #{druid_id} for target #{target} of type #{type} in #{sprintf('%0.4f', elapsed)}s"
     end
 
+    ##
+    # By convention we are using settings on sw-indexer that specify subtargets
+    # in all caps (thus the target upcase).
     # @param druid [String] represents the druid on the form of ab123cd4567
     # @param target_url [String] the url for the indexing service
     # @param solr_target [Array] solr target for solr core
     # @return [String] RestClient request command
     def build_request_url(druid, target_url, solr_target)
-      solr_target = { solr_target.upcase => true }
-      "#{target_url}/items/#{druid}?#{solr_target.to_query('subtargets')}"
+      "#{target_url}/items/#{druid}/subtargets/#{solr_target.upcase}"
```

Comment:
the param doc ^^^ says `solr_target` is an Array. the `.upcase` assumes it's a String. should it be `first.to_s.upcase` instead? or is the doc wrong